### PR TITLE
Use @actions/http-client

### PR DESCRIPTION
**Pull request by ericmj**

### CODE REVIEW

**Comment by ericmj** [comment]
@paulo-ferraz-oliveira Not sure why the gleam tests are failing. They are failing in previous commits also.

File: package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
   "dependencies": {
     "@actions/core": "1.10.0",
     "@actions/exec": "1.1.1",
+    "@actions/http-client": "^2.1.0",
```
Comment (paulo-ferraz-oliveira) [inline package.json:24]:
```suggestion
    "@actions/http-client": "2.1.0",
```

**Comment by paulo-ferraz-oliveira** [comment]
> They are failing in previous commits also.

Not sure either ("Uncaught error in rebar_core.") but not important in the context of this pull request, don't worry.

**Review by paulo-ferraz-oliveira — APPROVED** [review]
Approved with a very minor comment.

**Comment by paulo-ferraz-oliveira** [comment]
Merge at will if you have no further changes.

**Comment by paulo-ferraz-oliveira** [comment]
Released in `v1.16.0` / `v1.16`. Tag `v1` moved up...

**Pull request merged**